.drone.yml

```diff
@@ -86,6 +86,9 @@ pipeline:
       - PLATFORM=Linux
       - BEHAT_SUITE=${BEHAT_SUITE}
     commands:
+      - cd /var/www/owncloud
+      - su-exec www-data php occ config:system:set trusted_proxies 1 --value=`ip addr show eth0 | awk '$1 == "inet" {gsub(/\\/.*$/, "", $2); print $2}'`
```

Comment:
@individual-it do you want to keep the ``inet`` output?
I guess it does not hurt so in future when IP-based tests start failing we can see what the drone interfaces look like?

Reply:
for local drone I had to user `eth1` for drone-ci `eth0` so leaving both here